hack/lima-kubestellar.yaml

```diff
@@ -0,0 +1,179 @@
+- description: "wait for KubeFlex to be installed"
+  script: |
+    #!/bin/bash
+    set -eux -o pipefail
+    if ! timeout 30s bash -c "until command -v kflex >/dev/null 2>&1; do sleep 3; done"; then
+      echo >&2 "KubeFlex is not installed."
+      exit 1
+    fi
+- description: "wait for KubeFlex to be available"
+  script: |
+    #!/bin/bash
+    set -eux -o pipefail
+    if ! timeout 30s bash -c "until kubectl config use-context kind-kubeflex >/dev/null 2>&1; do sleep 3; done"; then
+      echo >&2 "KubeFlex is not available."
+      exit 1
+    fi
+- description: "wait for KubeStellar Core to be available"
+  script: |
+    #!/bin/bash
+    set -eux -o pipefail
+
+    if ! timeout 300s bash -c "until kflex ctx --overwrite-existing-context its1 >/dev/null 2>&1; do sleep 3; done"; then
+      echo >&2 "KubeStellar Core is not available."
+      exit 1
+    fi
+- description: "wait for managedcluster cluster1 to be available"
+  script: |
+    #!/bin/bash
+    set -eux -o pipefail
+
+    if ! timeout 300s bash -c "until kubectl --context its1 get managedclusters | grep -q cluster1 >/dev/null 2>&1; do sleep 3; done"; then
+      echo >&2 "managedcluster cluster1 is not available."
+      exit 1
+    fi
+
+- description: "wait for managedcluster cluster2 to be available"
+  script: |
+    #!/bin/bash
+    set -eux -o pipefail
+
+    if ! timeout 300s bash -c "until kubectl --context its1 get managedclusters | grep -q cluster2 >/dev/null 2>&1; do sleep 3; done"; then
+      echo >&2 "managedcluster cluster2 is not available."
+      exit 1
+    fi
+
+  hint: Use "sudo tail -f /var/log/cloud-init-output.log" in the guest to see progress.
+hostResolver:
+  # hostResolver.hosts requires lima 0.8.3 or later. Names defined here will also
+  # resolve inside containers, and not just inside the VM itself.
+  hosts:
+    host.docker.internal: host.lima.internal
+portForwards:
+- guestSocket: "/run/user/{{.UID}}/docker.sock"
+  hostSocket: "{{.Dir}}/sock/docker.sock"
+message: |
+  ------
+  host_context=kind-kubeflex
+  its_cp=its1
+  its_context=its1
+  wds_cp=wds1
+  wds_context=wds1
+  wec1_name=cluster1
+  wec2_name=cluster2
+  wec1_context=$wec1_name
+  wec2_context=$wec2_name
+  label_query_both=location-group=edge
+  label_query_one=name=cluster1
+
+  To access the KubeStellar demo environment from the host, run the following commands:
+    kubectl config delete-context cluster1
+    kind export kubeconfig --name cluster1
+    kubectl config rename-context kind-cluster1 cluster1
+    kubectl config delete-context cluster2
+    kind export kubeconfig --name cluster2
+    kubectl config rename-context kind-cluster2 cluster2
+    kubectl config delete-context kind-kubeflex
+    kind export kubeconfig --name kubeflex
+    kflex ctx --set-current-for-hosting
+    kflex ctx --overwrite-existing-context wds1
+    kflex ctx --overwrite-existing-context wds2
+    kflex ctx --overwrite-existing-context its1
+  ------
+mountType: virtiofs
+vmType: vz
```

Comment:
Is it necessary or desirable to specify this? https://lima-vm.io/docs/config/vmtype/ seems to say that it is not always the right choice.